### PR TITLE
Make compute_network_adapter resource importable

### PR DESCRIPTION
**Pull request by aureldsn**
Partially fix issue #12

### CODE REVIEW

File: examples/resources/cloudtemple_compute_network_adapter/import.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+$ terraform import cloudtemple_compute_network_adapter.foo c74060bf-ebb3-455a-b0b0-d0dcb79f3d86
```
Comment (remilapeyre) [inline examples/resources/cloudtemple_compute_network_adapter/import.sh:1]:
```suggestion
$ terraform import cloudtemple_compute_network_adapter.foo c74060bf-ebb3-455a-b0b0-d0dcb79f3d86

```

File: examples/resources/cloudtemple_compute_virtual_machine/import.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+$ terraform import cloudtemple_compute_virtual_machine.foo de2b8b80-8b90-414a-bc33-e12f61a4c05c
```
Comment (remilapeyre) [inline examples/resources/cloudtemple_compute_virtual_machine/import.sh:1]:
```suggestion
$ terraform import cloudtemple_compute_virtual_machine.foo de2b8b80-8b90-414a-bc33-e12f61a4c05c

```

**Review by remilapeyre — CHANGES_REQUESTED** [review]
There can only be one network adapter per network and the ID of the network adapter is always the same than the one of the network?

**Comment by aureldsn** [comment]
> There can only be one network adapter per network and the ID of the network adapter is always the same than the one of the network?

A virtual machine can have multiple network adapters with the same network ID

> Oh ok, I think I got it, the addition of the `network_id` attribute is unrelated to making `compute_network_adapter` importable, that is correct?

No, if I didn't made this change, after the `cloudtemple_compute_network_adapter` resource is imported when we run a tf plan, it indicates to add the `network_id` parameter instead of doing nothing

**Pull request merged**